src/lib.rs

```diff
@@ -59,9 +59,20 @@
 #![allow(non_camel_case_types)]
 #![allow(non_upper_case_globals)]
 #![warn(trivial_numeric_casts)]
+#![cfg_attr(micromath, no_std)]
```

Comment:
Here and below, you want `feature = "micromath"`, not just `micromath`. As-is this is always disabled.